neofetch

```diff
@@ -1087,6 +1087,22 @@ get_distro() {
 
             distro=$(trim_quotes "$distro")
             distro=${distro/NAME=}
+
+            # Get Ubuntu flavor.
+            if [[ $distro == "Ubuntu"* ]]; then
+                for i in plasma mate xubuntu Lubuntu budgie ubuntustudio; do
+                    [[ $XDG_CONFIG_DIRS =~ $i ]] && flavor=$i && break
+                done
+
+                case $flavor in
+                    "plasma")  distro=${distro/Ubuntu/Kubuntu} ;;
+                    "mate")    distro=${distro/Ubuntu/Ubuntu MATE} ;;
+                    "xubuntu") distro=${distro/Ubuntu/Xubuntu} ;;
+                    "Lubuntu") distro=${distro/Ubuntu/Lubuntu} ;;
+                    "budgie")  distro=${distro/Ubuntu/Ubuntu Budgie} ;;
+                    "ubuntustudio") distro=${distro/Ubuntu/Ubuntu Studio} ;;
+                esac
```

Comment:
Does something like this not work?

```
case $XDG_CONFIG_DIRS in
    *plasma*) ;;
    ...
esac
```

Reply:
Ah, yes that looks nicer and should also work. I will try it, thanks.